components/Snackbar/src/private/MDCSnackbarOverlayView.m

```diff
@@ -412,7 +412,7 @@ - (CGFloat)maximumHeight {
 }
 
 - (BOOL)anchoredToScreenEdge {
-  return self.bottomOffset == 0;
+  return [self dynamicBottomMargin] == 0;
```

Comment:
Is it possible to add a unit test for the bug? I'm going to guess we at least need to do some subclassing with exposed methods. If not, at least we have the demo, but it would be good if we can detect regressions in the future.

Reply:
thanks, i will open an issue to look into this further when time permits